### ISSUE TITLE
How to bind dataset with the sectionIndicator?

### ISSUE THREAD

**Issue by HanderWei** [danoz73/RecyclerViewFastScroller#43]
Any help will be highly appreciated!

**Comment by AkshayChordiya** [comment]
Yeah documentation is very weak

**Comment by ItsCalebJones** [comment]
Has anyone had luck with this yet?

**Comment by kkamaraj** [comment]
I agree on the documentation issue. I used the sample app as reference to bind dataset with section indicator. 
**Steps as follows,**
1. Implement the interface _SectionIndexer_ on your RecyclerView dataset Adapter. Both getSections() and getSectionForPosition() methods are called from recyclerviewfastscroller module.
2. Build an Enum on Section Indexer and getSections() method to return the enum values as array of Objects
3. The Core logic to return the section indexer would on the method getSectionForPosition(int position) where you would return the index for the enum values which would be used to fetch from array returned from getSections()
